### PR TITLE
display git merge group first

### LINKED ISSUE #5528: [git] merge group should be the first one
Right now, it is the last:
Screen Shot 2019-06-19 at 17 34 39

### PR DESCRIPTION
**Pull request by akosyakov**
- fix #5528

#### How to test

- create a project
- reproduce merge conflict in it https://stackoverflow.com/questions/43702944/reproducing-git-merge-conflict-dd
- check that merge group is first when the project is opened in Theia

### CODE REVIEW

**Comment by vince-fugnitto** [comment]
@kittaakos should we merge for the release since @akosyakov is on vacation?

**Comment by kittaakos** [comment]
> @kittaakos should we merge for the release since @akosyakov is on vacation?

Sure, why not. The code looks good, there are two approvals. Please go ahead.

File: packages/git/src/browser/git-scm-provider.ts [hunk 1]
```diff
@@ -151,15 +151,15 @@ export class GitScmProvider implements ScmProvider {
                 }
             }
         }
-        state.groups.push(await this.createGroup('index', 'Staged changes', state.stagedChanges, true));
+        state.groups.push(await this.createGroup('merge', 'Merged Changes', state.mergeChanges, true));
```
Comment (AlexTugarev) [inline packages/git/src/browser/git-scm-provider.ts]: `Merged Changes` -> `Merge Changes` would be even better.

Comment (kittaakos) [reply packages/git/src/browser/git-scm-provider.ts]: Makes sense: https://github.com/microsoft/vscode/blob/30379ab74b74f76b94c28872cac3a9b1a38ff82e/extensions/git/src/repository.ts#L676

Comment (AlexTugarev) [reply packages/git/src/browser/git-scm-provider.ts]: thanks for checking!

**Pull request merged**